Update product.rst (#4848)
mattermost/docs: source/about/product.rst

Core committers, including community contributors and paid staff at Mattermost, Inc., determine the project roadmap. For enterprises with needs beyond the scope of the open source project, commercial "Enterprise Edition" extensions are available from Mattermost, Inc. Partnership with our core committer community, along with revenue from Enterprise Edition, ensures the continued improvement of all editions.

Mattermost Team Edition
-----------------------

A free-to-use, open source, self-hosted alternative to proprietary SaaS messaging. Team Edition is your open source "virtual office", offering all the core productivity benefits of competing SaaS solutions. It deploys as a single Linux binary with MySQL or PostgreSQL under an MIT license.

The open source Mattermost Team Edition is functionally identical to the commercial Mattermost Enterprise Edition in its free “team mode”, but there is no ability to unlock enterprise features.

> We generally recommend installing Enterprise Edition, even if you don't currently need a license. This provides the flexibility to unlock Enterprise features seamlessly should you need them. However, if you only want to install software with a fully open source code base, then Team Edition is the best choice for you.

Reviewer: ```suggestion
We generally recommend installing Enterprise Edition, even if you don't currently need a license. This provides the flexibility to seamlessly unlock Enterprise features should you need them. However, if you only want to install software with a fully open source code base, then Team Edition is the best choice for you.
```

Mattermost Enterprise Edition E20
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> A commercial enterprise-grade messaging system that scales from hundreds to tens of thousands of users. E20 is your "virtual city", with cluster-based horizontal scaling, high availability deployment, performance monitoring systems and compliance reporting, plus advanced corporate directory integration. Building on the features in E10, it also offers:

Reviewer: ```suggestion
A commercial enterprise-grade messaging system that scales from hundreds to tens of thousands of users. Enterprise Edition E20 is your "virtual city", with cluster-based horizontal scaling, high availability deployment, performance monitoring systems and compliance reporting, plus advanced corporate directory integration. Building on the features in Enterprise Edition E10, it also offers:
```

Mattermost Cloud delivers the industry’s leading open-source collaboration platform in a highly secure, dedicated cloud environment designed for privacy-conscious enterprises and developer collaboration.

Mattermost Cloud is built on open-source software powered by a thriving developer community. The open-source approach enables software with a modular architecture built with modern software development practices. Open-source development and provides transparency in source code, extensibility, and public trust. Mattermost Cloud comes in two editions: Mattermost Cloud Professional and Mattermost Cloud Enterprise.

Mattermost Cloud Professional
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Mattermost Cloud Professional includes the complete set of collaboration and DevOps features, available from the Mattermost.com website. Signing up requires creating an account with an email and specifying a domain for the workspace. Once you sign up and create your workspace, you're guided through a quick setup including invitating team members and integrating with developer tools. You can trial Mattermost Cloud Professional for free for 14 days.

Reviewer: ```suggestion
Mattermost Cloud Professional includes the complete set of collaboration and DevOps features, available from the Mattermost.com website. Signing up requires creating an account with an email and specifying a domain for the workspace. Once you sign up and create your workspace, you're guided through a quick setup including inviting team members and integrating with developer tools. You can trial Mattermost Cloud Professional for free for 14 days.
```

> Mattermost Cloud Professional includes the complete set of collaboration and DevOps features, available from the Mattermost.com website. Signing up requires creating an account with an email and specifying a domain for the workspace. Once you sign up and create your workspace, a wizard will guide you through a quick setup, invitation for team members, and integration with developer tools. You can trial Mattermost Cloud Professional for free for 14 days.

Reviewer: ```suggestion
Mattermost Cloud Professional includes the complete set of collaboration and DevOps features, available from the Mattermost.com website. Signing up requires creating an account with an email and specifying a domain for the workspace. Once you sign up and create your workspace, you're guided through a quick setup including invitating team members and integrating with developer tools. You can trial Mattermost Cloud Professional for free for 14 days.
```

Reviewer: Thank you!

Mattermost Cloud Enterprise
~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Mattermost Cloud Enterprise offers the isolation, security, and control of self-hosted editions but without the burden of managing deployment, maintaining uptime, or applying upgrades. Mattermost Cloud Enterprise offers all chat and administration features, integrated DevOps workflows, provides 99.0% financially-backed guaranteed uptime and enterprise-grade support.

Reviewer: ```suggestion
Mattermost Cloud Enterprise offers the isolation, security, and control of self-managed editions but without the burden of managing deployment, maintaining uptime, or applying upgrades. Mattermost Cloud Enterprise offers all chat and administration features, integrated DevOps workflows, and provides 99.0% financially-backed guaranteed uptime and enterprise-grade support.
```

How much does Mattermost Cloud cost?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Mattermost Cloud Professional is $10 USD per user per month as a pay-as-you-go subscription. If you have over 1000 users, please contact us.

Reviewer: ```suggestion
Mattermost Cloud Professional is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users, please contact us.
```

What are the alternatives, and why should customers choose Mattermost Cloud?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Other SaaS offerings are closed-source and focus on a very broad use-case. Mattermost Cloud is built on our open source core offering, allowing customers to view our source code and have a high degree of transparency into Mattermost development processes. Mattermost provides a DevOps-centric offering and has deep integrations and workflows that help builders and operators be more effective.

Reviewer: ```suggestion
Other SaaS offerings are closed-source and focus on a very broad use case. Mattermost Cloud is built on our open source core offering, allowing customers to view our source code and have a high degree of transparency into Mattermost development processes. Mattermost provides a DevOps-centric offering and has deep integrations and workflows that help builders and operators be more effective.
```
